frontend/app/(defaultLayout)/layout.tsx

```diff
@@ -9,14 +11,16 @@ export default async function DefaultLayout({
 }: {
    children: ReactNode;
 }) {
+   const readonlyHeaders = headers();
```

Comment:
I'm not entirely sure if this is the only place, but the only reason we're using `headers()` here is to know if we're being proxied in development so we can use relative API urls (i.e. to `{user}.cominor.com`)

This functionality was added in #1025. It is only needed in development and we could definitely turn this (extracting ifixit-origin from host header) on and off via an env variable if that would help.

Reply:
I think that's a fair point, though we'll want to be careful about having different behavior in dev/prod.